packages/gatsby/src/cache-dir/static-entry.js

```diff
@@ -195,11 +188,6 @@ export default (pagePath, callback) => {
     pathPrefix,
   })
 
-  const runtimeRaw = fs.readFileSync(
-    join(process.cwd(), `public`, runtimeScript),
-    `utf-8`
-  )
-
```

Comment:
this isn't currently used anywhere - should it be used?

Reply:
This looks ok to delete - the related code was removed in https://github.com/gatsbyjs/gatsby/commit/7983e271a7d27a029108508edd64b45c5466c17e#diff-917ba78a52f29b1a1fe42be34d81fc83